### ISSUE TITLE
Validation loss doesn't decrease (驗證損失不會下降)

### ISSUE THREAD

**Issue by AdrienGuo** [ohhhyeahhh/SiamCAR#91]
When training, I found although the training loss is decreasing, the validation loss doesn't decrease.
But the latter model is indeed better than the former.
Does anyone face the same weird phenomenon?

我在訓練時發現，雖然訓練損失有在下降，但是驗證損失卻幾乎沒下降，
但奇怪的是，訓練越多次的 model 的確比前面的 model 表現更好？
有人也遇到類似的奇怪現象嗎？

要怎麼解釋這種情況呢？

**Comment by Kogel9701** [comment]
我也觉得训练时打出来的loss很奇怪，不降，一直在一个数徘徊。而且我想知道输出的loss里，括号里的loss是代表什么意思？

**Comment by JMChenl** [comment]
> 
这个代码没有验证损失吧？
